server.js

```diff
@@ -1,23 +1,69 @@
 import express from 'express'
 import bodyParser from 'body-parser'
 import cors from 'cors'
+import mongoose from 'mongoose'
+
+import books from './data/books.json'
 
-// Defines the port the app will run on. Defaults to 8080, but can be
-// overridden when starting the server. For example:
-//
-//   PORT=9000 npm start
 const port = process.env.PORT || 8080
 const app = express()
 
+const mongoUrl = process.env.MONGO_URL || "mongodb://localhost/livesession-mongo"
+mongoose.connect(mongoUrl, { useNewUrlParser: true, useUnifiedTopology: true})
+mongoose.Promise = Promise
+//adding any fun comment
 // Add middlewares to enable cors and json body parsing
 app.use(cors())
 app.use(bodyParser.json())
 
+const Book = mongoose.model('Book', {
+  bookID: Number,
+  title: String,
+  authors: String,
+  average_rating: Number,
+  isbn: Number,
+  isbn13: Number,
+  language_code: String,
+  num_pages: Number,
+  ratings_count: Number,
+  text_reviews_count: Number,
+})
+
+
+if (process.env.RESET_DB) {
+  const seedDatabase = async () => {
+    await Book.deleteMany({});
+
+    books.forEach((item) => {
+      const newBook = new Book(item);
+      newBook.save();
+    });
+  };
+  seedDatabase();
+}
+
 // Start defining your routes here
 app.get('/', (req, res) => {
   res.send('Hello world')
 })
 
+//getting all the books
+app.get('/books', async (req, res) => {
+  console.log(req.query)
```

Comment:
Great that you are using console.logs, but remember to remove them when it is time for deployment